cypress-tests/cypress/e2e/PaymentMethodListTest/00000-PaymentMethodListTests.cy.js

```diff
@@ -316,7 +316,7 @@ describe("Payment Method list using Constraint Graph flow tests", () => {
         let res_data = data["Response"];
 
         cy.createPaymentIntentTest(
-          create_payment_body_with_currency_country("EUR", "US"),
+          create_payment_body_with_currency_country("EUR", "US", "US"),
```

Comment:
I have never seen a use case where we pass billing country and shipping country separate and they always remained the same.  
I would suggest you to revert this change.

Reply:
We have this scenario everywhere , this is one of the use cases for every merchant , so i added this scenario

Reply:
Since we've hard coded the value to be `US` for both (billing and shipping), why not go with a single variable?

Reply:
We have a case where we need to add different values